models/DataObject/Data/CalculatedValue.php

```diff
@@ -64,7 +64,7 @@ public function __construct($fieldname)
      *
      * @param string $ownerType
      * @param string|null $ownerName
-     * @param int|null $index
+     * @param int|string|null $index
```

Comment:
Should be also changed for the attribute and the getter return

Reply:
You are way too fast 💯 
And I am really looking forward to having everything typed.